### PR TITLE
Team Summary v1.0 Docs changes

### PR DESCRIPTION
**Pull request by akhilkohlimicrosoft**

### CODE REVIEW

**Review by shantanu1singh — APPROVED** [review]
LGTM

File: api-reference/v1.0/api/team-get.md [hunk 1]
```diff
@@ -131,6 +131,11 @@ Content-type: application/json
   },
   "discoverySettings": {
     "showInTeamsSearchAndSuggestions": true
+  },
+  "summary": {
+    "ownersCount": 2,
+    "membersCount": 3,
+    "guestsCount": 4,
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/team-get.md:138]: Please apply this same update to the create and update operations.

File: api-reference/v1.0/resources/team.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ For more information about working with groups and members in teams, see [Use th
 |messagingSettings|[teamMessagingSettings](teammessagingsettings.md) |Settings to configure messaging and mentions in the team.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/team.md:63]: Line 42 of this document: I noticed the **Complete migration** method in the table says it returns a team object, however, the method topic says it returns a 204 No Content. Would you please verify and update accordingly? If it is the table that needs the fix, please say "None" in the Return type column.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please add the new property to the examples in the create and update methods as well and fix a minor inconsistency with the Complete migration method. 

Kind regards,

Isaac

File: api-reference/v1.0/resources/team.md [hunk 3]
```diff
@@ -63,6 +63,7 @@ For more information about working with groups and members in teams, see [Use th
...
 |webUrl|string (readonly) | A hyperlink that will go to the team in the Microsoft Teams client. This is the URL that you get when you right-click a team in the Microsoft Teams client and select **Get link to team**. This URL should be treated as an opaque blob, and not parsed. |
 |createdDateTime|dateTimeOffset|Timestamp at which the team was created.|
+|summary|[teamSummary](teamsummary.md)| Contains summary information about the team, including number of owners, members, and guests. |
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/team.md:66]: Please order this property table alphabetically.

**Comment by isvargasmsft** [comment]
Implementation delayed as per author's clarification.

**Pull request closed without merging**